src/mscorlib/Common/PinnableBufferCache.cs

```diff
@@ -21,22 +21,6 @@ namespace System
 {
     internal sealed class PinnableBufferCache
     {
-        /// <summary>
-        /// Create a new cache for pinned byte[] buffers
-        /// </summary>
-        /// <param name="cacheName">A name used in diagnostic messages</param>
-        /// <param name="numberOfElements">The size of byte[] buffers in the cache (they are all the same size)</param>
-        public PinnableBufferCache(string cacheName, int numberOfElements) : this(cacheName, () => new byte[numberOfElements]) { }
```

Comment:
@brianrob can you clarify? I can't ifdef anything out because Overlapped is using it?